patches/@cypress+unique-selector+0.4.4.patch

```diff
@@ -0,0 +1,293 @@
+diff --git a/node_modules/@cypress/unique-selector/lib/index.esm.js b/node_modules/@cypress/unique-selector/lib/index.esm.js
+new file mode 100644
+index 0000000..60a443e
+--- /dev/null
```

Comment:
Can we just fix an release a new cypress unique selector instead of patch package?

Reply:
As you can see below, errors are flooding.